### PR TITLE
milestone-1: Mobile version skeleton

### PR DESCRIPTION
**Pull request by jenadiusnicholaus**
Mobile version skeleton:
   In this milestone added
- > Index.html file
- > small-screen.css file
- >wide-screen.css file which will used in future to design desktop version of it.
- > Updated ReadMe file
- > Designed mobile version skeleton
- >  and main,js in case we will need to use in future

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -5,12 +5,41 @@
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Hello microverse</title>
-    <link rel="stylesheet" href="index.css" />
+    <link rel="stylesheet" media="screen and (min-width: 600px)" href="assets/css/smallscreen.css">
+    <link rel="stylesheet" media="screen and (max-width: 600px)" href="assets/css/widecreen.css">
```
Comment (thecodechaser) [inline index.html]: - [ ] Your style is not working, Please remove the media attribute from your linked stylesheets. Media quarries must be used in CSS files.

File: index.html [hunk 2]
```diff
@@ -5,12 +5,41 @@
...
+        <h5 >I'm a Software Developer.</h5>
+        <p>I can help you build a product , feature or website Look through some of my work and experience! If you like what you
+        see and have a project you need coded,
+        don't hestiate to contact me.</p>
+        <div class="link-container">
+            <a href="#" class="links"><i class="fa-brands fa-github"></i></a>
+            <a href="#" class="links"><i class="fa-brands fa-linkedin-in"></i></a>
+            <a href="#" class="links"><i class="fa-brands fa-twitter"></i></a>
+            <a href="#" class="links"><i class="fa-brands fa-facebook-f"></i></a>
+            <a href="#" class="links"><i class="fa-brands fa-reddit-alien"></i></a>
```
Comment (thecodechaser) [inline index.html]: - [ ] Please wrap your social media links in a list, You can use `<ul>` list. When we have similar elements in HTML it's a good practice to wrap them in a list and it also makes sure our code is semantic.

File: assets/css/smallscreen.css [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+    flex-direction: row;
+    display: flex;
+    justify-content: space-between;
+    flex-wrap: nowrap;
+    align-items: center;
+  }
+
+  .header nav .logo {
+    font-size: 20px;
+  }
```
Comment (thecodechaser) [inline assets/css/smallscreen.css]: - [x] Good job working on the styles but still the header is a little different from the requirements. You are supposed to stick to the [figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) as much as possible. Please fix the size of your header elements heading and hamburger icon and make sure it matches the design.

Your design

Screenshot (55)


Figma design

Screenshot (56)

File: assets/css/smallscreen.css [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+    max-width: 100vw;
+    margin: 0;
+    padding-top: 20px;
+    padding-left: 20px;
+    padding-right: 20px;
+    background-color: #1c1a19;
+    background-repeat: no-repeat;
+    background-attachment: fixed;
+    background-position: bottom;
+    background-image: url('https://lh5.googleusercontent.com/proxy/w3oh8vPkBYObX9gqYJpOXMA-lraeW32SsT_6fLv032xO3OwQBeKWRWzovY9c-CLwONIsT4R3VTeZAWeDUg0f2NAYh0SEWONVpJWf-mtClY6BcbPIZKd2TEQ=s0-d');
```
Comment (thecodechaser) [inline assets/css/smallscreen.css]: - [ ] Please remove this background image from the hero section and replace it with the vector images on the Figma templates and make sure your design matches the template.

Your design

Screenshot (57)

Figma design

Screenshot (58)

**Review by thecodechaser — CHANGES_REQUESTED** [review]
## Hi @jenadiusnicholaus 👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎉
- No linter errors. :heavy_check_mark:
- GitHub flow is followed. :heavy_check_mark:
- PR has a title and summary. :heavy_check_mark:
- Code is clean. :heavy_check_mark:
- Professional README file. :heavy_check_mark:

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- [ ] Please use camel case for your commit messages
Bad Example: "add HTML file"
Good Example: "Add HTML file" 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: assets/css/smallscreen.css (hunk 4 above)
Comment (jenadiusnicholaus) [reply assets/css/smallscreen.css]: Thanks for your comment

File: index.html [hunk 5]
```diff
@@ -5,12 +5,43 @@
...
-    <h1 class='hello-microvers-title'>Hello microverse</h1>
+    <!-- Header -->
+    <div class="header">
+        <a href="#" class="logo">My Logo</a>
+        <a href="#" class="icon" >
+            <i class="fa-solid fa-bars"></i>
+        </a>
+    </div>
+    <!-- end of header -->
+    <!-- landing page -->
```
Comment (elmar8287) [inline index.html:22]: - [ ] [OPTIONAL] Please, try to remove comments, to make your code more clear 👍

**Review by elmar8287 — APPROVED** [review]
Hi @jenadiusnicholaus, 

# APPROVED  👍 🥇 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

Congratulations! 🎉 

Please, find my OPTiONAL suggestion in the comments under the review 👍 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 5 above)
Comment (jenadiusnicholaus) [reply index.html:22]: Thanks for you comment and approval

**Pull request merged**